content/en/docs/tasks/tools/_index.md

```diff
@@ -8,34 +8,42 @@ no_list: true
 ## kubectl
 
 The Kubernetes command-line tool, `kubectl`, allows you to run commands against
-Kubernetes clusters. You can use kubectl to deploy applications, inspect and manage
-cluster resources, and view logs.
+Kubernetes clusters. You can use kubectl to deploy applications, inspect and
```

Comment:
nit: looks like the back ticks were dropped around kubectl

Reply:
oops, good catch, thx I'll fix that up.

Reply:
Should "kind" and "minikube" also get the back ticks treatment throughout? `kind` looks to have back ticks on it's linked page.

Reply:
This looks like it should apply to both "kind" and "minikube," so while correcting this one, I've added back ticks around the other two throughout.